fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/Statistics.java

```diff
@@ -32,17 +32,23 @@ public double getOutputRowCount() {
 
     public double getOutputSize(ColumnRefSet outputColumns) {
         double totalSize = 0;
+        boolean nonEmpty = false;
         for (Map.Entry<ColumnRefOperator, ColumnStatistic> entry : columnStatistics.entrySet()) {
             if (outputColumns.contains(entry.getKey().getId())) {
                 totalSize += entry.getValue().getAverageRowSize();
+                nonEmpty = true;
             }
         }
+        if (nonEmpty) {
+            totalSize = Math.max(totalSize, 1.0);
+        }
         return totalSize * outputRowCount;
     }
 
     public double getComputeSize() {
-        return this.columnStatistics.values().stream().map(ColumnStatistic::getAverageRowSize).
-                reduce(0.0, Double::sum) * outputRowCount;
+        // Make it at least 1 byte, otherwise the cost model would propagate estimate error
+        return Math.max(1.0, this.columnStatistics.values().stream().map(ColumnStatistic::getAverageRowSize).
```

Comment:
Why column statistics average rowsize could less than 1 byte？

Reply:
A column's row-size could be zero, when all rows are null.

In this case, the statistic of JSON is calculated wrongly, and it's 0.